proposed/http-handlers/request-handlers.md

```diff
@@ -83,17 +84,16 @@ use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\ServerRequestInterface;
 
 /**
+ * Handles a server request and produces a response
+ *
  * An HTTP request handler process a HTTP request and produces an HTTP response.
```

Comment:
Two things on this line (yes, I realize you didn't change it!):

- need to consistently use the `an HTTP` phrasing, not `a HTTP`.
- I'd rephrase `and produces` to `in order to produce`.